### PR TITLE
Add tabs component

### PR DESCRIPTION
**Pull request by alexander-schranz**
Like the [accordion](https://github.com/sulu/web-js/pull/61) the Tab component is build on top of wacg aria attributes for accessible access:

https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html

Fiddle: https://jsfiddle.net/Lfyawrm3/

### CODE REVIEW

**Comment by alexander-schranz** [comment]
This component works and could be used this way. But currently I want to check first if it is implemented following the accessibility rules for such a component.

**Comment by alexander-schranz** [comment]
After having a look at the w3c recommendation we should orientate here by [W3C Aria Example Tabs with Automatic Activation](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html) or [W3C Aria example Tab with Manual Activation](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html) and build on top of aria and hidden attributes to force accessibility of the component.

**Comment by alexander-schranz** [comment]
This should be refractord to remove jquery and build on top of w3c accessible tabs attributes: https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html.

Example the buttons can then be selected the following way:

```js
tabs.buttons = el.querySelector('[role=tablist]').querySelectorAll('[role=tab]');
```

Similar to the accordion component: https://github.com/sulu/web-js/pull/61

File: packages/components/tabs/tabs.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+     *             aria-selected="true"
+     *             aria-controls="nils-tab"
+     *             id="nils">
+     *       Nils Frahm
+     *     </button>
+     *     <button role="tab"
+     *             aria-selected="false"
+     *             aria-controls="agnes-tab"
+     *             id="agnes"
+     *             tabindex="-1">
```
Comment (luca-rath) [inline packages/components/tabs/tabs.js:25]: Why setting `tabindex` to `-1` for second and last element but not for first one?

Comment (luca-rath) [reply packages/components/tabs/tabs.js:25]: It's not possible to switch tabs using keyboard currently

Comment (alexander-schranz) [reply packages/components/tabs/tabs.js:25]: This is 1:1 the same example from the https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html

Tested it there. It seems also not focusable on tab, but its switchable over the arrow keys. Will Change that.

Comment (alexander-schranz) [reply packages/components/tabs/tabs.js:25]: Keyboard accessiblity is updated.

File: packages/components/tabs/tabs.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+                event.preventDefault();
+                event.stopPropagation();
+
+                tabs.toggle(item);
+            });
+
+            item.button.addEventListener('keydown', (event) => {
+                var newIndex = null;
+
+                if (event.keyCode === 37 || event.keyCode === 38) { // Left key and up key
```
Comment (luca-rath) [inline packages/components/tabs/tabs.js:98]: `event.keyCode` is deprecated, see https://developer.mozilla.org/en/docs/Web/API/KeyboardEvent/keyCode ... I guess you should use `event.key` instead

Comment (luca-rath) [reply packages/components/tabs/tabs.js:98]: Ignoring, as `event.keyCode` has better browser support

**Pull request merged**